### PR TITLE
feat: improve non-initialized pages

### PR DESCRIPTION
**Pull request by VikaCep**
When the plugin is not initialized, we now display dedicated pages according to each route. 

Home
image

Checks
image

Probes
image

Alerts
image

Config
image
image

Fixes https://github.com/grafana/synthetic-monitoring-app/issues/807

### CODE REVIEW

**Comment by mem** [comment]
This makes sense to me. Thanks!

I'll defer to someone else to do the actual review of the code.

**Review by ckbedwell — CHANGES_REQUESTED** [review]
A few very minor nitpicky things but otherwise this is ace! 👏 

I've noticed with my local set-up I have some buggyness with app initialisation that you get no UI feedback on but that can be addressed in a separate ticket which I'll expand upon.

I also noticed a bug with the native card component when there is no link provided which I'll also address in a separate PR shortly.

The main problem areas I spotted are on the Config page:

Screenshot of Synthetic Monitoring plugin with the access token modal open, however the modal is empty of any content except a title that says 'Access Token'

You can press the `Generate access token` button but the modal displays it as empty.

Screenshot of Synthetic Monitoring plugin with the terraform config modal open. It shows an error at the top 'Cannot read properties of undefined (reading 'listProbes'). It also shows three code blocks. The bottom two related to importing existing checks and probes are empty.

If I open the Terraform Config I get an error at the top and the existing code blocks are empty. I think we should hide these being displayed at all.

In regards to the tf.json code block, we should also make a note that they would need to initialize the plugin for it to work?

Screenshot of the get started card for Alerts in Synthetic Monitoring on an ultra-wide monitor. The whole card stretches across the screen.

We should put a max-width on these cards, too.

Screenshot of Synthetic Monitoring plugin on the checks page. It shows a subtitle under the title.

We should remove the subtitle on the checks page for consistency, too. None of the other pages show a subtitle.

File: src/hooks/useAppInitializer.ts [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    metricsByUid,
+    logsByName,
+    logsByUid,
+    initialize,
+    datasourceModalOpen,
+    setDataSouceModalOpen,
+  };
+};
+
+export default useAppInitializer;
```
[the remark this answers is not in the document]
Comment (VikaCep) [reply src/hooks/useAppInitializer.ts]: I agree with you, and great idea to enforce this in a eslint rule for consistency. Created a PR for that: https://github.com/grafana/synthetic-monitoring-app/pull/843

**Review by ckbedwell — APPROVED** [review]
LGTM 💪 🔥

**Pull request merged**